src/shared/lib/oql/AccessorsForOqlFilter.ts

```diff
@@ -32,6 +32,22 @@ export type SimplifiedMutationType =
     | 'inframe'
     | 'other';
 
+export function getMutationSubType(d: {
+    mutationType: string;
+    proteinChange: string;
+}) {
+    if (d.mutationType && d.mutationType.toLowerCase() === 'fusion') {
```

Comment:
i feel like these exist somewhere in enums

Reply:
This code is not new, I just extracted it from `public mut_type(d: Datum)` to be able to reuse it. Most of the string references in this file is hardcoded already. If we really want to fix this specific fusion reference only, I think we can use this one: https://github.com/cBioPortal/cbioportal-frontend/blob/10928633533e2c55c9f4530cf545af5e3f11da74/packages/cbioportal-frontend-commons/src/lib/getCanonicalMutationType.ts#L16